File: chapter06/aakkademaid-scala-6/src/main/scala/com/lp/akka/notes/MainApp.scala
package com.lp.akka.notes

import akka.actor.{ActorSystem, Props}

/**
 * @title TODO
 * @author li.pan
 * @version 1.0.0
 * @createTime 2021年04月30日 14:42:00
 *             <p>
 *             $description
 *             </p>
 */
object MainApp extends App {
  val system = ActorSystem("Akkademy")

  val clusterController = system.actorOf(Props[ClusterController], "clusterController")
}

File: chapter06/aakkademaid-java-6/src/main/java/com/lp/akka/notes/MainApp.java
package com.lp.akka.notes;

import akka.actor.ActorRef;
import akka.actor.ActorSystem;
import akka.actor.Props;
import akka.contrib.pattern.ClusterReceptionistExtension;
import akka.routing.BalancingPool;

/**
 * Actor系统启动入口
 * 可以通过-Dakka.remote.netty.tcp.port=2551 在同一机器上启动多个实例。（指定为 0，由 Akka 来随机分配一个端口）
 */
public class MainApp {
    public static void main(String... args) {
        ActorSystem system = ActorSystem.create("Akkademy");
        ActorRef clusterController = system.actorOf(Props.create(ClusterController.class), "clusterController");

    }
}
